spec/lib/task-graph-spec.js

```diff
@@ -520,6 +520,14 @@ describe('Task Graph', function () {
                     .that.equals('finished');
             });
         });
+
+        it('should create task with waitOn.either', function(){
```

Comment:
Just a question, I actually have my own test at hand, but I am wondering, how can I merge/upload it to our existing test system, should I use FIT or CIT?